### PR TITLE
update docs to include info about bcc, cc, and message id for email and msg partitioning

### PR DESCRIPTION
**Pull request by Coniferish**
See https://github.com/Unstructured-IO/unstructured/pull/3446

### CODE REVIEW

**Comment by Paul-Cornell** [comment]
Pulled branch locally, built locally, and saw no obvious rendering errors.

**Review by Paul-Cornell — APPROVED** [review]

File: open-source/core-functionality/partitioning.mdx [hunk 1]
```diff
@@ -189,7 +189,18 @@ For more information about the `partition_docx` function, you can check the [sou
...
+
+| Header     | Element Type | Metadata Name    |
+| ---        | ---          | ---              |
+| Subject    | Subject      | subject          |
+| From       | Sender       | sent_from        |
+| To         | Recipient    | sent_to          |
+| Cc         | Recipient    | cc_recipient     |
+| Bcc        | Recipient    | bcc_recipient    |
+| Received   | ReceivedInfo | NA               |
+| Message-ID | MetaData     | email_message_id |
```
Comment (Paul-Cornell) [inline open-source/core-functionality/partitioning.mdx:202]: Just double-checking, is this `metadata`, `MetaData`, `Metadata`, or something else?

File: open-source/core-functionality/partitioning.mdx [hunk 2]
```diff
@@ -189,7 +189,18 @@ For more information about the `partition_docx` function, you can check the [sou
...
+The `partition_email` function partitions `.eml` documents and works with exports from email clients such as Microsoft Outlook and Gmail. The `partition_email` takes a filename, file-like object, or raw text as input and produces a list of document `Element` objects as output. Also `content_source` can be set to `text/html` (default) or `text/plain` to process the html or plain text version of the email, respectively. In order for `partition_email` to return the header information as elements, `include_headers` must be set to `True`. Header information is captured in element metadata regardless of the `include_headers` setting.
+
+| Header     | Element Type | Metadata Name    |
```
Comment (Paul-Cornell) [inline open-source/core-functionality/partitioning.mdx:194]:
```suggestion
| Header     | Element type | Metadata name    |
```

File: open-source/core-functionality/partitioning.mdx (hunk 1 above)
Comment (Coniferish) [reply open-source/core-functionality/partitioning.mdx:202]: There's are a couple of element types specific to emails that we use. `MetaData` is one of them: https://github.com/Unstructured-IO/unstructured/blob/0f057188c6eda11cdc7e169b1332d01328622409/unstructured/documents/email_elements.py#L94

**Comment by Paul-Cornell** [comment]
@Coniferish OK'd me to merge this one.

**Pull request merged**